x-pack/plugins/triggers_actions_ui/public/application/lib/alert_api/common_transformations.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { AlertExecutionStatus } from '../../../../../alerting/common';
+import { RewriteRequestCase } from '../../../../../actions/common';
+import { Alert, AlertAction } from '../../../types';
+
+export const transformAction: RewriteRequestCase<AlertAction> = ({
+  group,
+  id,
+  connector_type_id: actionTypeId,
+  params,
+}) => ({
+  group,
+  id,
+  params,
+  actionTypeId,
+});
+
+export const transformExecutionStatus: RewriteRequestCase<AlertExecutionStatus> = ({
+  last_execution_date: lastExecutionDate,
+  ...rest
+}) => ({
+  lastExecutionDate,
+  ...rest,
+});
+
+export const transformAlert: RewriteRequestCase<Alert> = ({
```

Comment:
Would it make sense for `transformAlert` to call `transformExecutionStatus` and `transformAction` as part of its transformation?